test/run_test.py

```diff
@@ -98,7 +94,10 @@
         DISTRIBUTED_TESTS_CONFIG['nccl'] = {
             'WORLD_SIZE': '2' if torch.cuda.device_count() == 2 else '3'
         }
-
+    if dist.is_gloo_available():
+        DISTRIBUTED_TESTS_CONFIG['gloo'] = {
+            'WORLD_SIZE': '2' if torch.cuda.device_count() == 2 else '3'
```

Comment:
Would I be correct if I assume this would only work for `test_distributed.py` not `test_c10d.py`?

Reply:
Yes, that's correct. The tests in `test_c10d.py` check the process group availability themselves, typically on a per class basis.